fixtures/ios_app_with_multi_configs/App/Project.swift

```diff
@@ -11,9 +11,10 @@ let settings = Settings(base: [
 ], configurations: configurations)
 
 let betaScheme = Scheme(name: "App-Beta",
-                         shared: true,
-                         buildAction: BuildAction(targets: ["App"]),
-                         runAction: RunAction(configurationName: "Beta", executable: "App"))
+                        shared: true,
+                        buildAction: BuildAction(targets: ["App"]),
+                        runAction: RunAction(configurationName: "Beta", executable: "App"),
+                        archiveAction: ArchiveAction(configurationName: "Beta"))
```

Comment:
should we update a fixture test to ensure the archive scheme can execute successfully?

Reply:
Yes, it needs to be updated, how do I run the fixtures with my version of tuist? Or is another thing that needs to change?

Reply:
Our fixtures are defined in [`fixtures/`](https://github.com/tuist/tuist/tree/master/fixtures) and are exexuted by our [cucumber test's](https://github.com/tuist/tuist/blob/master/features/generate.feature).

To run them locally you would execute [`features`](https://github.com/tuist/tuist/blob/master/Rakefile#L13) from the Rakefile:
```sh
bundle exec rake features
```

Inside the generate feature you should be able [to specify which action](https://github.com/tuist/tuist/blob/master/features/step_definitions/shared/xcode.rb#L6) you would like Xcode to run as part of your scheme.

An example of this would be you can define a step in the [cucumber tests](https://github.com/tuist/tuist/blob/master/features/generate.feature#L182) as follows:
```
Then I should be able to archive for iOS the scheme App
```

If you are unsure of any of the above or would like me to step in then please ask

Reply:
Sorry, but I don't know how this actually works. I wrote a new step as you suggested, but I'm getting the error `error: use of unresolved identifier 'ArchiveAction'` when a run `bundle exec rake features`, I guess it is because the tests aren't being run with my version of tuist.